src/docs/sdk/performance/ui-event-transactions.mdx

```diff
@@ -0,0 +1,166 @@
+        UI element
+    Then the SDK reschedule the idle timeout
+    And doesn't create a new transaction
+
+# If your SDK binds auto-generated transactions to the scope see binding
+# to scope
+Scenario: Same UI element with different event
+    Given an ongoing UI event transaction
+    When the user triggers the same UI element with a different event
+    Or the user triggers a different UI element
+    Then the SDK finishes the ongoing transaction
+    And sets the status to OK
+    And waits for the children to finish
+    And cancels the idle timeout
+    And starts a new transaction
+```
+
+## Binding to scope
+
+On platforms mainly interacting with the static API, such as mobile, it's typical to bind auto-generated
+transactions to the scope so that users can access them via the static API. We recommend binding UI event
+transactions to the scope on these platforms. The following extra rules apply as UI event transactions
+could interfere with other auto-generated transactions.
+
+```Gherkin
+Scenario: Same UI element with different event
+    Given an ongoing UI event transaction
+    When the user triggers the same UI element with a different event
+    Or the user triggers a different UI element
+    Then the SDK finishes the ongoing transaction
+    And sets the status to OK
+    And waits for the children to finish
+    And cancels the idle timeout
+    And removes the ongoing transaction from the scope
+    And starts a new transaction
+    And puts the new transaction on the scope
+
+Scenario: UI event triggered but transaction ended
+    Given an auto-generated transaction from a any UI event
+    And the transaction already  is finished
+    When the user triggers the same UI event
+    Then the SDK starts a new UI event transaction
+
+# The SDK adds auto-generated spans to the transaction bound to the scope.
+# UI event transactions need auto-generated spans not to be empty. Therefore,
+# the SDK can't start a UI event transaction if there is already a
+# transaction bound to the scope by the user.
+Scenario: Manually created transaction bound to the scope
+    Given an ongoing manually created transaction by the user bound to the
+        scope
+    When the user triggers the a UI event
+    Then the SDK doesn't start a UI event transaction
+```
+
+
+### Screen load transactions
+
+This section deals specifically with auto-generated transactions for loading screens. If your SDK has
+other types of auto-generated transactions, please update the specification here.
+
+```Gherkin
+Scenario: Ongoing UI event transaction
+    Given an ongoing UI event transaction
+    When the SDK creates a new screen load transaction
+    Then the SDK finishes the ongoing UI event transaction
+    And removes it from the scope
+    And sets the status to canceled
+    And waits for its children to finish
+
+Scenario: Ongoing screen load transaction
+    Given an ongoing screen load transaction
+    When the SDK starts a new UI event transaction
```

Comment:
@philipphofmann I guess we should also change it to `does not start a new UI event transaction` similar to Manually create transactions